backend/src/test/java/com/ambacam/ItBase.java

```diff
@@ -1,45 +1,50 @@
 package com.ambacam;
 

```

Comment:
Please pay attention to formatting

Reply:
It will be difficult to pay attention to the formatting for @Coccoonx and @MistraForest since they use IntelliJ. What I propose is, while we are looking for a corresponding formatting file for IntelliJ or a convertion, one of us (@Chombon and I) will reformat the code in the branch before merging it.